unicore/comments/service/views/comments.py

```diff
@@ -86,5 +102,54 @@ def delete_comment(request, uuid, connection):
 
 
 '''
-TODO: Comment collection resource
+Comment collection resource
 '''
+
+
+@app.route('/comments/', methods=['GET'])
+@inlineCallbacks
+def list_comments(request):
+    columns = Comment.__table__.c
+    filter_expr = comment_filters.get_filter_expression(request.args, columns)
```

Comment:
Ahhh, cool, that should be fine. I was scared it was just accepting the query string params raw.